### DISCUSSION TITLE
Request to create a branch 5.0.0 for feature test.

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Ed/Community#64] [Installation] [answered]
Hello all,

The Sunbird Video Player needs to be tested as a new feature of Interactive Video is coming in. 
[Details Here](https://project-sunbird.atlassian.net/browse/SB-30516)
We need to have a branch for the same to test the package .
Please create a branch 5.0.0 so that we can raise a PR for the updated package. 

@coolbung @baljeetsingh @vinukumar-vs @harishgangula

**Comment by vinukumar-vs** [comment, marked as answer]
@Xitija  Can you create the PR for release-4.10.2 itself? 

Don't forget to add the comments in PR as we discussed.

@sharathkashyap @luckynvdu Please help them to verify this feature in dev environment. The content creation will happen in VDN & comption need to verify in portal.

**Reply by Xitija** [reply]
I have raised the PR as discussed .
[https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8210](https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8210)
Please let me know if anything needs to be added.

**Reply by sharathkashyap** [reply]
@Xitija Kindly go ahead with the integration with current release branch to test your changes. However, the actual delivery can be done as part of 5.0 release itself. You may have to revert the library package before 4.10.3 goes to staging regression.

**Reply by Xitija** [reply]
@sharathkashyap @luckynvdu 
Please merge this PR
[ https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8210]( https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8210)
We are in need to test this and sign off this by today.

**Reply by rajeevsathish** [reply]
@Xitija as the release-4.10.2 is already out from ED post QA verification you need to  create the pr for next release branch

CC:@sharathkashyap

**Reply by Xitija** [reply]
@rajeevsathish 
Sir can you please tell me where can I raise this PR then? Which is the next release branch?

cc: @coolbung @baljeetsingh @vinukumar-vs @HarishGangula

**Reply by rajeevsathish** [reply]
@Xitija you can use release-4.10.3

**Reply by Xitija** [reply]
@rajeevsathish 
Sir I have updated the PR to so that it can be merged in branch-4.10.3
Please merge it .  [https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8210](https://github.com/Sunbird-Ed/SunbirdEd-portal/pull/8210)
also please let us know when the PR will be merged where can we see the changes on
dev.sunbirded.org or staging.sunbirded.org ?

cc :  @coolbung @baljeetsingh @vinukumar-vs @HarishGangula @sharathkashyap @luckynvdu

**Reply by sharathkashyap** [reply]
@Xitija Kindly let us know if you would like to take this thread further. 
cc: @luckynvdu  @rajeevsathish

**Reply by Xitija** [reply]
@sharathkashyap 
Thankyou so much for your concern sir.
We got the branch and sent the PR.
I do not want to take this thread further .

**Comment by sharathkashyap** [comment]
@Xitija Locking this conversation.
